setup.py

```diff
@@ -20,9 +20,9 @@ def runSetup():
     Calls setup(). This function exists so the setup() invocation preceded more internal
     functionality. The `version` module is imported dynamically by importVersion() below.
     """
-    boto = 'boto==2.38.0'
-    boto3 = 'boto3>=1.4.7, <2.0'
-    futures = 'futures==3.1.1'
+    boto = 'boto==2.48.0'  # Previously 'boto==2.38.0'
+    boto3 = 'boto3>=1.7.50, <2.0'  # Previously 'boto3==1.4.7'
+    futures = 'futures==3.2.0'  # Previously 'futures==3.1.1'
```

Comment:
Keep this as `futures==3.1.1` since that is the latest version in python 3.

Reply:
Its been reverted